app/views/audits/edit.html.erb

```diff
@@ -0,0 +1,63 @@
+<section class="content-header">
+  <% content_for :title, "Edit - Audits - Inventory - #{current_organization.name}" %>
+  <h1>
+    Edit Inventory Audit
+    <small>for <%= current_organization.name %></small>
+  </h1>
+  <ol class="breadcrumb">
+    <li><%= link_to(dashboard_path) do %>
+        <i class="fa fa-dashboard"></i> Home
+      <% end %>
+    </li>
+    <li><%= link_to "Inventory Audits", (audits_path) %></li>
+    <li class="active"> New Audit</li>
+  </ol>
+</section>
+
+<!-- Main content -->
+<section class="content">
+
+  <!-- Default box -->
+  <div class="box">
+    <div class="box-header with-border">
+      <p class="box-title">Edit the <strong>Audit</strong> recording all the inventory present in a particular storage location.</p>
+    </div>
+
+    <div class="box-body">
+      <p class="help">Enter <code>all</code> the items with their corresponding <code>quantity</code> that are present in the storage location</p>
+      <p class="help">Click on <code>Save Progress</code> button to save the current progress of audit.</p>
+      <p class="help">You can <code>resume</code> the saved audit at a later time again.</p>
```

Comment:
Please replace the `<code>` tags with a semantically-correct tag -- you're welcome to use the same formatting in the CSS, if you like -- if you use a tag like `<em>` instead, you could do something like
```css
.help > em {
  /* the styles */
}```
and that'd keep it from affecting anything else.

Reply:
Ok, I will use `em` instead.